Mapify/Patches/ItemLocationForcerPatch.cs

```diff
@@ -0,0 +1,39 @@
+﻿using DV;
+using DV.Shops;
+using HarmonyLib;
+using Mapify.Components;
+using Mapify.Map;
+using VLB;
+
+namespace Mapify.Patches
+{
+    /// <summary>
+    /// Something keeps setting the shop scanner inactive. This patch prevents that.
```

Comment:
We should find what's causing this, not add some weird hacky workaround. You should be able to see what's happening by inspecting the stacktrace from the OnEnable/OnDisable methods of something attached to an item.